canvas_modules/common-canvas/src/common-properties/actions/button/button.jsx

```diff
@@ -17,11 +17,11 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { connect } from "react-redux";
-import Button from "carbon-components-react/lib/components/Button";
+import { Button } from "carbon-components-react";
```

Comment:
This imports the es modules instead of the commonJs which then allows for tree shaking.  Without doing this all the carbon-react icons are imported.